css/style.css

```diff
@@ -10,6 +10,23 @@ main {
   min-height: 100vh;
 }
 
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.button {
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  height: 24px;
+  padding: 12px;
+  width: fit-content;
+}
+
 .button:hover {
   background-color: #ff6b00;
 }
```

Comment:
- [ ] Please add a hover and pressed state on your buttons to match the design template. You can use the `:hover` and `:active` pseudo-classes for this.
![f2](https://user-images.githubusercontent.com/94551484/192039904-e44f6759-af17-4a2e-b4ef-e3474cf7181b.png)